src/libraries/System.Collections.Immutable/src/System.Collections.Immutable.csproj

```diff
@@ -23,6 +23,9 @@ System.Collections.Immutable.ImmutableSortedSet&lt;T&gt;
 System.Collections.Immutable.ImmutableStack
 System.Collections.Immutable.ImmutableStack&lt;T&gt;</PackageDescription>
   </PropertyGroup>
+  <ItemGroup>
+    <ProjectReference Include="$(LibrariesProjectRoot)System.Runtime.CompilerServices.Unsafe\src\System.Runtime.CompilerServices.Unsafe.ilproj" />
```

Comment:
Please move this section down. P2Ps are always defined after Compile/None items.